### PR TITLE
Feature/final models endpoints

### PR DESCRIPTION
**Pull request by MichaelHabermas**
# Description

all endpoints up, tho not rich

Details of your changes

PREPARE

Why?

## Type of change

- New feature (non-breaking change which adds functionality)?
- Bug Fix?

## Change Status

- Completed, ready to review and merge?
- In progress?
- etc.

## Checklist

- [ ] Endpoint tested with Postman, or Unit test.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -18,17 +18,27 @@ const swaggerUIOptions = {
 };
 
 //###[  Routers ]###
-const indexRouter = require('./index/indexRouter');
-const providerRouter = require('./provider/providerRouter');
-const programRouter = require('./program/programRouter');
-const statusRouter = require('./statuses/statusesRouter');
```
Comment (ahmedseragcodes) [inline api/app.js:24]: I don't see a statusRouter added down below, is this something that still needs to be made or just gone completely?

File: api/ethnicities/ethnicityModel.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const findById = async (id) => {
+  return await knex('ethnicities').where('ethnicity_id', id).first();
+  // keeping these for now as a reference
+  // .select(knex.raw('ethnicities.*, to_json(locations.*) as location'))
+  // .groupBy('ethnicities.ethnicity_id', 'locations.location_id');
+};
+
+const createEthnicity = async (newEthnicity) => {
+  return await knex('ethnicities').insert(newEthnicity, ['*']);
```
Comment (ahmedseragcodes) [inline api/ethnicities/ethnicityModel.js:15]: Not sure what the ['*'] after the newEthnicity obeject is here?

**Review by ahmedseragcodes — APPROVED** [review]
Outside of what I commented about, changes make sense according to updates/plans, approved

File: api/ethnicities/ethnicityModel.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+const knex = require('../../data/db-config');
+
+const findAll = async () => {
+  return await knex('ethnicities');
+};
+
+const findById = async (id) => {
+  return await knex('ethnicities').where('ethnicity_id', id).first();
+  // keeping these for now as a reference
+  // .select(knex.raw('ethnicities.*, to_json(locations.*) as location'))
```
Comment (ryan-hamblin) [inline api/ethnicities/ethnicityModel.js:10]: I see that this code has been commented out. Can we please remove it as per our standards?

File: api/genders/genderModel.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+const knex = require('../../data/db-config');
+
+const findAll = async () => {
+  return await knex('genders');
+};
+
+const findById = async (id) => {
+  return await knex('genders').where('gender_id', id).first();
+  // keeping these for now as a reference
+  // .select(knex.raw('genders.*, to_json(locations.*) as location'))
```
Comment (ryan-hamblin) [inline api/genders/genderModel.js:10]: Please remove commented out code. The code comment looks good.

**Review by ryan-hamblin — COMMENTED** [review]
there is a lot of work there that I think warrants a variety of pull requests. Maybe let's break the work out into sizeable chunks next time and focus on solving a single problem.

**Pull request merged**